guides/source/debugging_rails_applications.md

```diff
@@ -746,24 +771,22 @@ To list all active catchpoints use `catch`.
 There are two ways to resume execution of an application that is stopped in the
 debugger:
 
-* `continue [line-specification]` \(or `c`): resume program execution, at the
-address where your script last stopped; any breakpoints set at that address are
-bypassed. The optional argument line-specification allows you to specify a line
-number to set a one-time breakpoint which is deleted when that breakpoint is
-reached.
-* `finish [frame-number]` \(or `fin`): execute until the selected stack frame
-returns. If no frame number is given, the application will run until the
-currently selected frame returns. The currently selected frame starts out the
-most-recent frame or 0 if no frame positioning (e.g up, down or frame) has been
-performed. If a frame number is given it will run until the specified frame
-returns.
+* `continue[ n]`: resumes program execution at the address where your script last
```

Comment:
Hi @deivid-rodriguez , lets change to `continue [n]`, since we follow it elsewhere, as well as in this guide itself.
Example: `edit [file:n]`

Please change all occurrence in this guide.